client/src/stories/5-SideMenu.stories.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { MemoryRouter as Router } from 'react-router-dom';
+import SideMenu from '../components/Common/Navigation/SideMenu';
+import USER_TYPES from '../constants/userTypes';
+import * as N from '../constants/navRoutes';
+
+export default {
+  title: 'Menu',
+};
+
+const dummyLogOutFunction = () => {};
+
+export const SideMenuIntern = () => (
+  <Router initialEntries={[N.DASHBOARD_URL]}>
+    <SideMenu
+      role={USER_TYPES.intern}
+      isLoggedIn
+      resetState={dummyLogOutFunction}
+    />
+  </Router>
+);
+
+export const SideMenuHost = () => (
+  <Router initialEntries={[N.DASHBOARD_URL]}>
+    <SideMenu
+      role={USER_TYPES.host}
+      isLoggedIn
+      resetState={dummyLogOutFunction}
```

Comment:
instead of mocking functions, you can use storybook actions
```suggestion
      resetState={action('signed-out')}
```
you can get actions from storybook like this
`import { action } from '@storybook/addon-actions';`